### PR TITLE
[RA2 Ch04, RI2 Ch02,04] Update references and configuration options

### PR DESCRIPTION
**Pull request by michaelspedersen**
* Updates list of references from RA2
* Updates Kuberef configuration options to match current version

### CODE REVIEW

**Comment by michaelspedersen** [comment]
I am considering if the table in section 2.2 can be done differently.
* I am considering if we can remove the "requirement" part
  * it is a copy/paste from RA2, and any changes to the text will have to me mirrored manually
* I have only copied requirements that RI2 currently fulfills to some extend
  * Alternatively the entire list can be copied, and we can add a new row that indicates the status of the requirement in RI2

I am also wondering where additional details from an implementation perspective can be added. Currently my best bet would be chapter 4, but I think it needs a bit of restructuring for that to make much sense.

**Comment by michaelspedersen** [comment]
From weekly meeting (5/12):
* Remove the requirement/reference table from RI2 Ch2
* Utilize the existing table in RA2 Ch4 to keep track of RI2 progress
  * There is already a column for traceability in RI2 (needs updating)
  * Consider adding another column that highlights if each requirement is supported in RI2

**Comment by michaelspedersen** [comment]
Changing this PR to draft until additional updates have been made

**Comment by michaelspedersen** [comment]
Traces have been updated in RA2, and the requirement text (copy/paste) has been removed from RI2. The entire table in RI2 could be removed, but I have left it for now as it has the profile details that otherwise would require additional referencing between RI2 and RA2

File: doc/ref_arch/kubernetes/chapters/chapter04.md [hunk 1]
```diff
@@ -74,14 +74,14 @@ In order for the Kubernetes components to be conformant with the Reference Archi
 |Ref|Specification|Details|Requirement Trace|Reference Implementation Trace|
 |---|---|---|---|---|
 |`ra2.k8s.001`|Kubernetes Conformance|The Kubernetes distribution, product, or installer used in the implementation must be listed in the [Kubernetes Distributions and Platforms document](https://docs.google.com/spreadsheets/d/1uF9BoDzzisHSQemXHIKegMhuythuq_GL3N1mlUUK2h0/edit#gid=0) and marked (X) as conformant for the Kubernetes version that is being used.|[req.gen.cnt.03](./chapter02.md#23-kubernetes-architecture-requirements)||
```
Comment (gkunz) [inline doc/ref_arch/kubernetes/chapters/chapter04.md:76]: If I am not mistaken, the underlying installer of BMRA is Kubespray, which is in fact listed. Worth a comment in the table?

Comment (michaelspedersen) [reply doc/ref_arch/kubernetes/chapters/chapter04.md:76]: The reason it wasn't included was that Kubespray wasn't certified for K8s 1.19 - But I can see it got the certification about an hour before you added this comment, so I will update the table :)

Comment (gkunz) [reply doc/ref_arch/kubernetes/chapters/chapter04.md:76]: oh, I didn't notice the timing! Thank you.

**Review by gkunz — APPROVED** [review]

**Review by rihabbanday — APPROVED** [review]

**Comment by pgoyal01** [comment]
@michaelspedersen This PR is changing Chpater 4 in RA2 and Chapter 2 in RI2. I think that should be made clear in the title; I have added the RA2 project under "Projects". Thus, it at least needs acceptance by @rgstori.

Many of the links have been changed from relative to absolute. I am not sure if it is possible to fix them now or an issue should be opened to fix them in the enxt release.

**Review by pgoyal01 — APPROVED** [review]
Issue should be opened to fix links -- make relative

**Pull request merged**